client/packages/system/src/Patient/ContactTrace/Components/columns.ts

```diff
@@ -11,55 +11,42 @@ import { ContactTraceRowFragment } from '@openmsupply-client/programs';
 export interface ContactTraceListColumnsProps {
   onChangeSortBy: (column: Column<ContactTraceRowFragment>) => void;
   sortBy: SortBy<ContactTraceRowFragment>;
-  includePatient?: boolean;
 }
 
 export const useContactTraceListColumns = ({
   onChangeSortBy,
   sortBy,
-  includePatient,
 }: ContactTraceListColumnsProps) => {
   const { localisedDate } = useFormatDateTime();
 
-  const columnList: ColumnDescription<ContactTraceRowFragment>[] = [];
-if (includePatient) {
-    columnList.push({
-      key: 'patientName',
-      label: 'label.patient',
-      accessor: ({ rowData }) => rowData?.patient?.name,
-      sortable: false,
-    });
-  }
-  columnList.push(
+  const columnList: ColumnDescription<ContactTraceRowFragment>[] = [
     {
       key: 'programName',
       label: 'label.program',
       accessor: ({ rowData }) => rowData.program.name,
       sortable: false,
     },
     {
-      key: 'contactTraceId',
-      label: 'label.contact_trace-id',
+      key: 'datetime',
+      label: 'label.date',
+      formatter: dateString =>
+        dateString ? localisedDate((dateString as string) || '') : '',
     },
     {
-      key: 'contactPatientName',
-      label: 'label.contact-patient',
-      accessor: ({ rowData }) => rowData?.contactPatient?.name,
-      sortable: false,
+      key: 'firstName',
+      label: 'label.first-name',
     },
     {
-      key: 'status',
-      label: 'label.status',
+      key: 'lastName',
+      label: 'label.last-name',
     },
     {
-      key: 'datetime',
-      label: 'label.date',
+      key: 'age',
+      label: 'label.age',
       align: ColumnAlign.Right,
       width: 175,
-      formatter: dateString =>
-        dateString ? localisedDate((dateString as string) || '') : '',
-    }
-  );
+    },
```

Comment:
The issue asked for `Gender` and `Contacted?` as well

Reply:
I added Gender but @richardmoizeau and I decited to leave "Contacted" out... (Planned to put it into the general contact tracing status but this would need some more thought...)